core-js/src/main/javascript/cdf/components/filter/HtmlUtils.js

```diff
@@ -0,0 +1,9 @@
+define(function(){
+  return {
+    sanitizeHtml: function(html){	  
+	  html = html.replace(/<script\b[^>]*>/g, "&lt;script&gt;").replace(/<\/script>/g, "&lt;/script&gt;");
+	  html = html.replace(/<iframe\b[^>]*>/g, "&lt;script&gt;").replace(/<\/iframe>/g, "&lt;/script&gt;");
```

Comment:
@carlosrusso I've done. But I believe <html> and <body> tags do not suppose injection.